fcrepo-clients/src/test/java/edu/unc/lib/dl/fcrepo4/FileObjectIT.java

```diff
@@ -38,42 +37,33 @@
 /**
  *
  * @author bbpennel
+ * @author harring
  *
  */
 public class FileObjectIT extends AbstractFedoraIT {
 
-    @Autowired
-    private Repository repository;
-
-    private PID pid;
-
     private static final String origBodyString = "Original data";
     private static final String origFilename = "original.txt";
     private static final String origMimetype = "text/plain";
     private static final String origSha1Checksum = DigestUtils.sha1Hex(origBodyString);
 
     @Before
     public void init() throws Exception {
-        // Generate a new ID every time so that tests don't conflict
-        pid = PIDs.get(RepositoryPathConstants.CONTENT_BASE + "/" + UUID.randomUUID().toString());
-
         createBaseContainer(RepositoryPathConstants.CONTENT_BASE);
+        repoObjLoader.init();
```

Comment:
I'd suggest calling this `init` method from the context file instead

Reply:
fixed